File: Exavision.Seasense.Shared/States/RecordingState.cs
using System;

namespace Exavision.Seasense.Shared.States {
    public class RecordingState : State {
        public string UserLogin { get; set; }
        public DateTime StartDate { get; set; }
        public string FileName { get; set; }

        public string Id { get; set; }

        public string MaterialId { get; set; }

        public string Description { get; set; }
    }
}

File: Exavision.Seasense/Client/src/app/materials/states/recording-state.ts
import { CapabilityState } from "./capability-state";
import { UnitState } from "./unit-state";

export interface RecordingState {
  $type: string;
  userLogin: string;
  startDate: Date;
  fileName: string;
  id: string;
  materialId: string;
  description: string;
  
}
